cps/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -359,7 +359,7 @@ private Env createInitialEnv() {
 
     private CpsScript parseScript() throws IOException {
         shell = new CpsGroovyShell(this);
-        CpsScript s = (CpsScript) shell.reparse("WorkflowScript",script);
+        CpsScript s = (CpsScript) shell.reparse("WorkflowScript", script);
```

Comment:
Please revert whitespace-only changes.

(BTW there is a filed issue to change this script name to something meaningful to the user, but it is a complex issue. And I am not entirely confident that any changes now are compatible for running builds.)